front/src/applications/editor/tools/pointEdition/utils.ts

```diff
@@ -18,7 +18,9 @@ export function getNewSignal(point?: [number, number]): SignalEntity {
     properties: {
       id: NEW_ENTITY_ID,
       extensions: {
-        sncf: {},
+        sncf: {
+          side: 'CENTER',
```

Comment:
Should we leave it with "LEFT" by default ?

Reply:
Both the [JSON schema](https://github.com/OpenRailAssociation/osrd/blob/66b26ab16650cef15baf6a40fa2ed0fba4ed317b/front/src/reducers/osrdconf/infra_schema.json#L1125) and [editoast](https://github.com/OpenRailAssociation/osrd/blob/66b26ab16650cef15baf6a40fa2ed0fba4ed317b/editoast/editoast_schemas/src/infra/side.rs#L16) have CENTER as the default, so I assumed this was the value that made the most sense from a UX point-of-view.